Panorama Public updates and bug fixes

vagisha opened the pull request:
#### Changes
- Add submitters to the "Panorama Public Submitters" group. Add reviewer to "Reviewers" group. These groups exist on panoramaweb.org.
- Replaced "Access URL / Link" with "Permanent Link".
- Added text about license (e.g. CC-BY) in the confirm view when data is made public.
- Fixed container links and "Make Public" link in notifications.  These have to be absolute URLs.
- Added DOI in the message body of the data-copied notification.
- Display DOI as a complete link as recommended by the DataCite DOI Display Guidelines (https://support.datacite.org/docs/datacite-doi-display-guidelines).

vagisha committed:
- Added text about license in confirm view when data is made public
- Add submitters to the "Panorama Public Submitters" group. Add reviewer to "Reviewers" group. These groups exist on panoramaweb.org.
- Fix container links and "Make Public" link in notifications.  These have to be absolute URLs

vagisha committed:
Replace "Access URL/Link" with "Permanent Link".

vagisha committed:
Fix tests

vagisha committed:
- Check that data license is not null.  Older datasets may not have a data license.
- Remove extra argument

labkey-jeckels commented on panoramapublic/src/org/labkey/panoramapublic/pipeline/CopyExperimentFinalTask.java:
```diff
@@ -524,11 +536,41 @@ private void updatePermissions(User formSubmitter, ExperimentAnnotations targetE
+    private void addToGroup(User user, String groupName, Container project, Logger log)
+    {
+        Integer groupId = SecurityManager.getGroupId(project, groupName, false);
+        if (groupId != null)
```
Would it be useful to log if we can't find the group? Does the test create it today?

vagisha replied:
Done.

vagisha committed:
- CR feedback: log if we cannot find the "Panorama Public Submitters" or "Reviewers" group.
- Updated test
- Added DOI in the message body of the data-copied notification.
- Display the DOI as a complete link as recommended by the DataCite DOI Display Guidelines.
- Added methods in DataCiteService and ProteomeXchangeService to return the URL for an identifier.
